src/pools/cluster/dynamic.ts

```diff
@@ -33,6 +38,11 @@ export class DynamicClusterPool<
     super(min, filename, opts)
   }
 
+  /**
+   * Choose a worker in a round robin fashion.
+   *
+   * If all active workers are currently busy, try creating a new worker and using it.
```

Comment:
```suggestion
   * Choose a worker for the next task.
   *
   * It will first check for and return an idle worker.
   * If all workers are busy, then it will try to create a new one up to the `max` worker count.
   * If the max worker count is reached, the emitter will emit a `FullPool` event and it will fall back to using a round robin algorithm to distribute the load.
```